assets/js/date-picker/date-picker.js

```diff
@@ -38,25 +116,46 @@ const datePicker = {
       maxViewMode: 0,
       datesDisabled,
       beforeShowDay: date => datePickerUtils.displayFirstOfMonth(date)
-    }).on('changeDate', event => {
-      datePicker.changeDateHandler(event);
-    });
+    }).on('changeDate', event => datePicker.changeDateHandler(event))
+      .on('destroy', event => console.info.bind(this, 'vabffabma'))
```

Comment:
@barbaracassani I don't think you meant to leave this here 😄

Reply:
oops 😊